Sources/TSCBasic/FileSystem.swift

```diff
@@ -552,9 +552,20 @@ public class InMemoryFileSystem: FileSystem {
     /// reality, the only practical use for InMemoryFileSystem is for unit
     /// tests.
     private let lock = Lock()
-    
     /// Exclusive file system lock vended to clients through `withLock()`.
```

Comment:
This comment should probably be updated since it's now a mapping from paths to locks again.

Reply:
done